src/main/scala/no/ndla/draftapi/service/WriteService.scala

```diff
@@ -470,6 +510,9 @@ trait WriteService {
           isImported = false,
           shouldAlwaysCopy = updatedApiArticle.createNewVersion.getOrElse(false)
         )
+        _ <- partialPublish(articleId,
+                            PartialArticleFields.values.toSeq,
+                            updatedApiArticle.language.getOrElse(Language.AllLanguages))._2
```

Comment:
Tror kanskje vi ikke skal ha `partialPublish` her, for dette er når det finnes ikke artikkelen i databasen fra før av mens id'en finnes(veldig edge-case). Så da opprettes en ny artikkel som har status `DRAFT` og med andre felter tilsendt fra api'et. 

Hvis artikkelen ikke var i databasen i draft kan den heller ikke være publisert, så dette vil returnere feil fra article-api isåfall. Så jeg tror at jeg bare fjerner dette herfra, med mindre jeg tenker feil?

Reply:
Dersom artikkelen ikkje finnes treng du ikkje delpublisering så då kan du sikkert fjerne dette.

Reply:
> Dersom artikkelen ikkje finnes treng du ikkje delpublisering så då kan du sikkert fjerne dette.

Vi klarte aldri finne ut hvorfor null-dokumentene finnes i databasen, så det _kan_ jo tenkes at de finnes i article-api men ikke i draft-api, og da burde den vel være her?
(Kan også hende at buggen ikke skjer lenger og hele denne funksjonen er useless, men det er litt vanskelig å vite da)

Reply:
Om du avpubliserer en artikkel så settes document i article-api til null. Kan det være det som er saka her?

Reply:
Tror ikke draft-api blir satt til `null` når vi avpubliserer hvis det var det du mente?
I guess det dukker opp en `null` rad når man prøver å opprette en ny artikkel med valideringsfeil (Som jo er uheldig, men egentlig ikke et problem i seg selv).

I teorien så trengs ikke dette kallet (eller denne funksjonen), så vi kan prøveå  fjerne den (hele funksjonen), så kan vi se om vi støter på samme problemet som vi hadde tidligere (som gjorde at denne funksjonen ble laget), mulig det har blitt løst siden vi ikke egentlig visste hva problemet var.